2024/calcite-react/demo/src/main.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App.tsx';
+
+import "./index.scss";
+import './App.scss';
+
+import { setAssetPath } from "@esri/calcite-components/dist/components";
+setAssetPath("https://cdn.jsdelivr.net/npm/@esri/calcite-components/dist/calcite/assets");
```

Comment:
(optional) should users alway use the CDN for assets or are there use cases for using local assets?
do we need to add a comment about that here/in the presentation or is it an uncommon use case?